uptane-whitepaper.md

```diff
@@ -1 +1,106 @@
+
+It’s time to acknowledge that the science fiction nightmares of the 20th century have become the real-world challenges of the 21st. The development and implementation of reliable, adaptable, and resilient protection for passenger vehicles and light trucks has never been more important. 
+
+This whitepaper describes Uptane, the first software update security system for the automotive industry intentionally developed to resist even attacks by nation-state level actors. It is designed in such a manner that the security of software updates does not degrade all at once, but follows a hierarchy in which different levels of access to vehicles or the automaker’s infrastructure must be gained. By building these levels into the security system, even if an attacker compromises servers, bribes operators, or gains access to vehicular networks, he or she is limited in how much danger they can cause. 
+
+## **Addressing the double-sided nature of software updates and why automobiles are so hard to secure** 
+Though not directed against automobiles, the recent [SolarWinds](https://en.wikipedia.org/wiki/SolarWinds)hack in which companies, government agencies, 
+and academic institutions suffered significant data breaches after malware was slipped into a software update, is a sobering reminder that though updates are necessary, they are also always fraught with risk.  The [full impact](https://en.wikipedia.org/wiki/2020_United_States_federal_government_data_breach) of the attack, which is known to have affected computer systems within the U.S. Departments of Defense, State, Homeland Security, Treasury, Commerce, and Energy, is still to be tallied.
+
+Yet, ignoring updates is not a viable option either  As Uptane steering committee member Dr. Justin Cappos, an associate professor of computer science and engineering, explained in a December 20 article in [Yahoo Finance](https://au.finance.yahoo.com/news/why-russias-massive-cyberattack-is-especially-insidious-222912267.html?__s=p54njaazgqic1gqfruk3), nation state actors are gravitating to such attack targets because updating software is something system maintainers are “supposed to be doing.” In a sense, OEMs and suppliers find themselves between the proverbial rock and a hard place. But, even though updates are risky, he cautioned,”If you don’t apply software updates, you’re absolutely, definitely vulnerable because old software is vulnerable software.”
```

Comment:
```suggestion
Yet, ignoring updates is not a viable option either. As Uptane steering committee member Dr. Justin Cappos, associate professor of computer science and engineering, explained in a December 20 article in [Yahoo Finance](https://au.finance.yahoo.com/news/why-russias-massive-cyberattack-is-especially-insidious-222912267.html?__s=p54njaazgqic1gqfruk3), nation-state actors are gravitating to such attack targets because updating software is something system maintainers are “supposed to be doing.” In a sense, OEMs and suppliers find themselves between the proverbial rock and a hard place. But, even though updates are risky, he cautioned, ”If you don’t apply software updates, you’re absolutely, definitely vulnerable because old software is vulnerable software.”
```